recipes/sunkit-magex/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "sunkit-magex" %}
+{% set version = "1.0.0a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sunkit_magex-{{ version }}.tar.gz
+  sha256: 606199962cef4d5c89c37a924caaf984314f8625999d84b0c95c3c3ee9fc9820
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.10
+    - setuptools >=62.1
+    - setuptools-scm >=6.2
+    - wheel
+    - pip
+  run:
+    - python >=3.10
+    - astropy >=5.2.0
+    - numpy >=1.23.0
+    - scikit-image >=0.19.0
+    - scipy >=1.8.0,!=1.10.0
+    - sunpy >=5.1
+    - streamtracer >=1.0.0
+
+test:
+  imports:
+    - sunkit_magex
+  commands:
+    - pip check
+    - pytest --doctest-rst -p no:unraisableexception -p no:threadexception --arraydiff
+  requires:
+    - pip
+    - pytest
+    - pytest-doctestplus
+    - pytest-arraydiff
+    - parfive
+    - reproject
+    - streamtracer
+    - sympy
+
+about:
+  summary: Solar Magnetic field Extrapolation
+  license: GPL-3.0
+  license_file: licenses/LICENSE.rst
+
+extra:
+  recipe-maintainers:
+    - jgieseler
+    - Cadair
+    - STBadman
```

Comment:
@STBadman please comment here to say you are onboard with co-maintaining this.

Reply:
I am on board with co-maintaining